### PR TITLE
Added logic to change title and copy for CF Everywhere

### PR DESCRIPTION
**Pull request by codingwithlis**
### About
Added logic to change title and copy for CF Everywhere

### Test Instructions
Test link -> /locations/cf-everywhere: you should see the changes on the What to Expect section  
These changes only apply to CF Everywhere, not any other campuses. 

### Screenshots
**Before**
Screenshot 2023-10-16 at 2 47 20 PM

**After**
Screen Shot 2023-10-16 at 3 59 05 PM

### Closes Tickets
[CFDP-2773](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2773)

### Related Tickets
N/A

[CFDP-2773]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2773?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by leoalfonso14 — DISMISSED** [review]
Looks good!

File: components/LocationSingle/locationData.js
Comment (dzwood) [inline components/LocationSingle/locationData.js:1]: What if we did this?
```
const whatToExpectData = name => {
  return {
    name,
    title: `What to Expect at ${name === 'Cf Everywhere' ? 'Christ Fellowship Everywhere' : name}`,
    htmlContent: `Here at Christ Fellowship Church in ${name === 'Cf Everywhere' ? 'Christ Fellowship Everywhere' : name}, you can expect to experience church services with uplifting worship music, encouraging messages from our pastors, special programming for your family, and opportunities for you to find people to do life with all throughout the week—it all starts here!`,
  };
};
```
Just to make it shorter without repeating the same text :)

**Review by dzwood — CHANGES_REQUESTED** [review]

File: components/LocationSingle/locationData.js
Comment (codingwithlis) [reply components/LocationSingle/locationData.js:1]: I thought about doing this but I changed the htmlContent copy too, I don't think "Here at Christ Fellowship Church in Christ Fellowship Everywhere" sounds right, but lmk what you think!

Comment (codingwithlis) [reply components/LocationSingle/locationData.js:1]: Got it!

**Review by dzwood — APPROVED** [review]
Looks awesome 🔥

**Pull request merged**